src/main/scala/org/globalforestwatch/layers/LitterCarbonEmisYear.scala

```diff
@@ -5,6 +5,8 @@ import org.globalforestwatch.grids.GridTile
 case class LitterCarbonEmisYear(gridTile: GridTile, model: String="standard")
   extends FloatLayer
     with OptionalFLayer {
-  val model_suffix = if (model == "standard") "" else s"__$model"
-  val uri: String = s"$basePath/gfw_litter_carbon_stock_in_emissions_year$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg/geotiff/${gridTile.tileId}.tif"
```

Comment:
Need to update layer path in datalake to ""$basePath/gfw_litter_carbon_stock_in_emissions_year$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg_ha-1/geotiff/${gridTile.tileId}.tif"
Change "gfw_litter_carbon_stock_in_emissions_year" to "gfw_litter_carbon_stock_in_emissions_year__standard" and Mg to Mg_ha-1.